policies/roadmap.md

```diff
@@ -0,0 +1,70 @@
+# OpenSSL project roadmap
+
+### Last modified 21 March 2022
+
+This document is the OpenSSL project roadmap. It is a living document
+and is expected to change over time. Objectives and dates should be
+considered aspirational.
+
+## Objectives
+
+Some of these objectives can be achieved more easily and quickly
+than others.
+
+#### Regular releases
+
+The OpenSSL project wants to move to having a regular release interval
+of approximately six months.
+
+#### QUIC
+
+OpenSSL is taking a multi-staged approach to the implementation of the
+[QUIC] transport protocol:
+
+- For OpenSSL 3.1, the focus is on providing a client side single stream
+QUIC implementation.
+
+- OpenSSL 3.2 will follow approximately six months later implementing more
+of the protocol.
+
+- OpenSSL 3.3 aims to complete the implementation a further six moths later.
+
+[QUIC]: https://datatracker.ietf.org/doc/html/rfc9000
+
+#### FIPS
+
+The OpenSSL FIPS provider for 3.x is still progressing through the validation.
+
+#### Post-quantum cryptography
+
+A cryptographic algorithm needs to be defined by a national or
+international standard before it will be considered for inclusion into
+OpenSSL.  Although, there is work in progress to select
+[post-quantum algorithms] for standardisation, currently none have been.
+OpenSSL will not be including any of the candidate algorithms until the
+selection process is complete.
+
+For those interested in using the proposed algorithms now, the
+[Open Quantum Safe] project has written a [provider] for OpenSSL 3.x
+which includes the candidates.
+
+[post-quantum algorithms]: https://csrc.nist.gov/Projects/post-quantum-cryptography
+[Open Quantum Safe]: https://openquantumsafe.org/
+[provider]: https://github.com/open-quantum-safe/openssl
```

Comment:
I have a different link:

https://github.com/open-quantum-safe/oqs-provider

Reply:
@mattcaswell is correct.  https://github.com/open-quantum-safe/openssl is a hacked 1.1.1, has nothing to do with providers

Reply:
Thanks.  I was mucking around trying to reference liboqs as well and after much fiddling decided just one URL and picked the wrong one naturally.